src/nunit-gui/Views/MainForm.cs

```diff
@@ -144,5 +144,10 @@ private void statusBarToolStripMenuItem_CheckedChanged(object sender, System.Eve
         {
             statusBarView.Visible = statusBarToolStripMenuItem.Checked;
         }
+
+        protected override void OnDragEnter(DragEventArgs drgevent){
+            if(drgevent.Data.GetDataPresent(DataFormats.FileDrop))
+                drgevent.Effect = DragDropEffects.Copy;
+        }
```

Comment:
ok, so yes you are right that you need both the visual effect and the subscribing to the event to capture getting the file names that are dropped, however that subscription is in the presenter. When we eventually move the SWF stuff out of the presenter we'll have to also override the DragDrop event to fire the new event in the View.

As a partial side note because I double checked there is extra stuff in the designer that I should delete. I'll do that and push up another commit